main.py

```diff
@@ -216,51 +231,18 @@ def parse_config():
 
     # check that required arguments exist
     if config_args.problem is None:
-        print('usage: ' + parser.prog + '\n' + parser.prog + ': error: argument -p/--problem: expected one argument')
+        print(one_arg_err(parser, '-p/--problem'))
         exit(1)
     if config_args.graph is None:
-        print('usage: ' + parser.prog + '\n' + parser.prog + ': error: argument -g/--graph: expected one argument')
+        print(one_arg_err(parser, '-g/--graph'))
         exit(1)
     if config_args.results is None:
-        print('usage: ' + parser.prog + '\n' + parser.prog + ': error: argument -r/--results: expected one argument')
-        exit(1)
-
-    # check that optional arguments are within their choices
```

Comment:
This is what my comment for 4. is referring to.

Reply:
Also, line `65-66` is redundant and can be removed, making line `67` an `if`.

Reply:
I didn't do anything to line 65-67.